SPECS/k3s/vendor_build.patch

```diff
@@ -1,40 +1,76 @@
-From bf8717d49b3b396d74096322b32186b3a8e6b3d9 Mon Sep 17 00:00:00 2001
-From: Lior Lustgarten <lilustg@microsoft.com>
-Date: Thu, 21 Jul 2022 13:20:43 -0700
-Subject: [PATCH 1/2] enable vendor build
-
----
- scripts/build       | 30 +++++++++++++++++++-----------
- scripts/package-cli |  2 +-
- 2 files changed, 20 insertions(+), 12 deletions(-)
-
+diff --git a/package/rpm/install.sh b/package/rpm/install.sh
```

Comment:
I think this patch does not apply cleanly.
We might need to update the patch.
The cause seems to be `fuzz-factor=0`

If you try to apply the patch locally, it will apply with a fuzz-factor=1

Reply:
Sure. Uploaded the new patch